components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -310,7 +310,10 @@ - (void)commonMDCFlexibleHeaderViewInit {
       (UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight);
   [super addSubview:_contentView];
 
-  self.backgroundColor = [UIColor lightGrayColor];
+  if (![MDCFlexibleHeaderView appearance].backgroundColor) {
```

Comment:
In effect we are setting the default backgroundColor to lightGrayColor.
Does it make sense to  [MDCFlexibileHeaderView appearance].backgroundColor = [UIColor lightGrayColor] in this classes +init method?
Client would be able to overwrite this stored value with their own explicit class to the appearance proxy.